lesson01/LESSON1.md

```diff
@@ -1,5 +1,41 @@
-### Setting up your tools
+### Checking your Computer
+Before any of the necessary software is downloaded, we need to make sure you have an appropriate computer that supports 
+our version of Minecraft and Forge. A good rule of thumb is start off with a computer that can run ordinary Minecraft pretty well.
 
-### IntelliJ
+Forge runs on Windows(7 and greater), and on Mac(10.8.3 and greater). However, all operating systems must be 64-bit. 
+Chromebooks are not supported.
 
-### Minecraft Plugin
\ No newline at end of file
+### Setting up your tools 
+#### Java
+We will be using Kotlin as our programming language, but Kotlin still generates Java bytecode when compiled. This means 
```

Comment:
We can probably leave anything mentioning bytecode out, just make it known that the Java Development Kit is necessary.